main.go

```diff
@@ -198,15 +201,19 @@ func main() { // nolint:gocyclo
 			}
 		}()
 	}
-
+	webhookServer := webhook.NewServer(webhook.Options{
+		Port: 9443,
+	})
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
+		Scheme: scheme,
+		Metrics: metricsserver.Options{
+			BindAddress: metricsAddr,
+		},
+		WebhookServer:          webhookServer,
```

Comment:
the same question as in the member-operator PR - do we need the webhookServer configured?